### PR TITLE
ovirt vm migrate with state present 

### PR DESCRIPTION
**Pull request by mnecas**
##### SUMMARY

ovirt vm migrate with state present

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1722403
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ovirt
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py [hunk 1]
```diff
@@ -1630,6 +1629,8 @@ def _attach_cd(self, entity):
 
     def _migrate_vm(self, entity):
         vm_host = self.param('host')
+        if entity.id is None and entity.name is not None:
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: Why is this condition needed?

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py [hunk 2]
```diff
@@ -2425,6 +2426,7 @@ def kernel_persist_check():
                         action_condition=lambda vm: vm.status == otypes.VmStatus.UP,
                         wait_condition=lambda vm: vm.status == otypes.VmStatus.UP,
                     )
+            vms_module._migrate_vm(vm)
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_vm.py:2428]: Please add a comment why we call it here.

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py (hunk 1 above)
Comment (mnecas) [reply lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: I had some issue with the entity but now I tested it now and looks fine without it

**Review by machacekondra — APPROVED** [review]
shipit

**Review by mwperina — APPROVED** [review]
shipit

**Comment by machacekondra** [comment]
shipit

**Pull request merged**